jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/impl/JettyClientUpgradeRequest.java

```diff
@@ -49,19 +49,16 @@ public JettyClientUpgradeRequest(WebSocketCoreClient coreClient, UpgradeRequest
         if (request != null)
         {
             // Copy request details into actual request
-            HttpFields fields = getHeaders();
+            HttpFields.Mutable fields = getHeaders();
             request.getHeaders().forEach(fields::put);
 
             // Copy manually created Cookies into place
             List<HttpCookie> cookies = request.getCookies();
             if (cookies != null)
             {
-                HttpFields headers = getHeaders();
                 // TODO: remove existing Cookie header (if set)?
                 for (HttpCookie cookie : cookies)
-                {
-                    headers.add(HttpHeader.COOKIE, cookie.toString());
-                }
+                    fields.add(HttpHeader.COOKIE, cookie.toString());
```

Comment:
Checkstyle will complain, as `for` always requires braces.

Reply:
Checkstyle is dead in intellij for me.  I've tried re-importing, but it just refuses to work?!?!?!?!